install/gnome.md

```diff
@@ -0,0 +1,96 @@
+# Install squeak-app in Gnome
+
+This guide describes a user-local installation.
+
+## Binaries
+
+- Download a compatible Linux bundle from <https://github.com/OpenSmalltalk/opensmalltalk-vm>
+
+- Extract its contents to `~/.local/squeak/<version>/`
+
+- Copy `balloon.svg` to `~/.local/squeak/squeak.svg`
+
+- Create `~/.local/bin/squeak`:
+
+  ```sh
+  #!/bin/sh
+  set -e
+  version=202502080249
+  cd "$HOME/.local/squeak/$version/"
+  exec ./squeak "$@"
```

Comment:
One issue with this script is that it can only be used as `squeak /absolute/path/to.image` but not as `squeak relative.image`. On the other hand, if we don't `cd` before the `exec`, the actual squeak script will not find the binaries. Not sure how to fix this...

Reply:
> Also, FileDirectory default and pwd never (?) were the same in Squeak ...

Or is this just the behavior I am seeing because of cd in the default Linux script?

Reply:
id say so, yes.

Reply:
> That's all Greek to me. :-)

What would be the output if you omitted the `cd` and used the full path to the squeak script instead?

Reply:
The right way to do this is to use $0, the name with which the startup script is executed, to locate the vm executable, which is relative in the bundle.

so one would say something like

“`dirname $0`/../lib/123456789/squeak” “$@“